packages/oncokb-frontend-commons/src/components/ReferenceList.tsx

```diff
@@ -54,24 +44,39 @@ export const ReferenceList: React.FunctionComponent<ReferenceListProps> = (
                 });
         });
 
-        return pubMedRecords[query];
+        const ret: any = [];
```

Comment:
Can we just return pubMedRecords?

Reply:
right, we should avoid `any` wherever possible.